example/adapters/bookshelf_postgresql_adapter.js

```diff
@@ -163,6 +163,7 @@ class BookshelfPostgresqlAdapter {
       .fetch();
     const data = record.get('data');
     data.consumed = getEpochTime();
+    data.consumedCount = 0;
```

Comment:
Nit: can we just remove this line if it's in an example?

Reply:
if this is an example, which is the real one we are using?